tests/xharness/Jenkins/TestTasks/BuildToolTask.cs

```diff
@@ -1,13 +1,14 @@
-﻿using System.Threading.Tasks;
+﻿using System;
+using System.Threading.Tasks;
 using Microsoft.DotNet.XHarness.iOS.Shared.Execution;
 
 namespace Xharness.Jenkins.TestTasks
 {
-	public abstract class BuildToolTask : AppleTestTask
+	public abstract class BuildToolTask: AppleTestTask
 	{
-		readonly Xharness.TestTasks.BuildToolTask buildToolTask;
+		protected Xharness.TestTasks.BuildToolTask buildToolTask;
 
-		public IProcessManager ProcessManager => buildToolTask.ProcessManager;
+		public IProcessManager ProcessManager { get; private set; }
```

Comment:
I think it's only set in ctor and then readonly property is enough.

```suggestion
		public IProcessManager ProcessManager { get; }
```